### ISSUE TITLE
Can TomoTwin simulate a full XPCI experiment?

### ISSUE THREAD

**Issue by Esquisilo** [aniketkt/TomoTwin#1]
Can TomoTwin simulate a source, x-ray spectrum, sample, and detector for an XPCI experiment, or does it mostly stick to the "Tomography" step?

**Comment by aniketkt** [comment]
Yes it's possible. So you're interested in generating projection images, right? The get_projections() method needs theta input. You can use theta = 0 for single projection.  For phase contrast, you can provide detector distance input and for defining your source, you may have to import some data from XOP software. Then you define the materials. If new material, their attenuation coefficients can be imported again from XOP. See the README file on how to import from XOP here. https://github.com/aniketkt/TomoTwin/tree/master/model_data

If you are adding more materials, feel free to do a pull request to add them to this repo directly.
